.circleci/config.yml

```diff
@@ -108,7 +101,7 @@ commands:
 
 # Job definitions
 jobs:
-  install:
+  setup:
```

Comment:
Changed `install` to `setup` to better reflect what the job does (checkout, rebase, install) and to match the name used in the FW repository.